README.md

```diff
@@ -4,18 +4,28 @@ packaged in Ubuntu 14.04). So all instructions below are based on 14.04 as the
 host:
 
 ```bash
-sudo apt-get install docker
+docker pull kartoza/geogig
+sudo docker run --name="geogig" -p 38080:8182 -d kartoza/geogig
+```
+
+
+If you want to build the image yourself using the Docker recipe then do the following:
+
+
+```bash
 sudo apt-get install apt-cacher-ng
-git clone git@github.com:timlinux/docker-projects.git
-cd docker-projects/geogit
+git clone git@github.com:kartoza/docker-geogig.git
+cd docker-geogig
 ```
 
 Edit ``71-apt-cacher-ng`` to use your host's ip address.
+Edit ``setup.sh`` to use your name and email address to configure geogig.
```

Comment:
Better if we can pass these details as env vars via docker run, then the docker image is independent of the username etc used